shared/src/main/kotlin/com/egm/stellio/shared/util/ApiUtils.kt

```diff
@@ -40,6 +41,15 @@ object ApiUtils {
         mapper.readValue(content, Notification::class.java)
 }
 
+fun generatesProblemDetails(response: ErrorResponse): String {
```

Comment:
Is it needed ? Default JSON serialization of an ErrorResponse does not fit the needs ?